mathics/builtin/atomic/atomic.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+"""Atomic Primitives
```

Comment:
The convention is to put `Atomic Primitives` in the next line.

Reply:
If I understand correctly, this should be addressed in 2a5ffcbe

BTW GNU Emacs Python mode prefers not to add a \n after a leading `"""`.

Reply:
It is as the other files now.